Makefile.rules_generic

```diff
@@ -68,8 +68,9 @@ prepare:
 
 BIN_TARGETS := bin/app.elf bin/app.apdu bin/app.sha256 bin/app.hex
 DBG_TARGETS := debug/app.map debug/app.asm
+MANIFEST := app.toml
 
-default: $(BIN_TARGETS) $(DBG_TARGETS)
+default: $(BIN_TARGETS) $(DBG_TARGETS) $(MANIFEST)
```

Comment:
Now that I've had a few issue with our app deployement piepeline, I'm not sure anymore we should add this to the default target. Even more if it needs custom python lib.

Reply:
Also, I'm not sure the MANIFEST vairable is really necessary.

Reply:
Ok